### PR TITLE
Feat/about content

### LINKED ISSUE #52: Write Mission and Inclusivity paragraphs

### LINKED ISSUE #53: Ensure accessibility for About page

### LINKED ISSUE #58: Add "our resources for this project" to About page

### LINKED ISSUE #60: Add "this data was last updated as of Sept 6, 2022"

### PR DESCRIPTION
**Pull request by irmakerdem**
** [Project Ticket](Project ticket link goes here) **
- Resolves #52 
- Resolves #53 
- Resolves #58 
- Resolves #60 

** Describe the functionality to be tested: **
- Add temporary mission and inclusivity statements
- Add statement about date of last update to data
- Ensures accessibility of About page with screen reader and tabbing
- Updates Cypress testing for mission, inclusivity, and data statements

** Giphy of how you feel submitting this PR: **

### CODE REVIEW

File: src/Views/AboutView/AboutView.js [hunk 1]
```diff
@@ -59,6 +59,10 @@ const AboutView = () => {
           <h3>Bryce W.</h3>
         </div>
       </div>
+      <div data-cy="data-statement" className="data-statement">
```
Comment (lkarpins) [inline src/Views/AboutView/AboutView.js:62]: Great addition, Irmak!

**Review by lkarpins — APPROVED** [review]

**Review by Rallen13 — APPROVED** [review]
Looks good to me, thanks for adding that info in!

File: cypress/e2e/aboutview.cy.js [hunk 2]
```diff
@@ -23,15 +23,15 @@ describe("About", () => {
...
+    cy.dataCy("mission-statement").contains("Our mission is to make abortion clinics information accessible to everyone in the US.");
     cy.dataCy("yes-illustration").should("be.visible");
   })
 
   it("Should display the inclusivity message", () => {
     cy.dataCy("inclusivity-title").contains("Inclusivity Counts");
-    cy.dataCy("inclusivity-statement").contains("Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.");
+    cy.dataCy("inclusivity-statement").contains("Abortion affects everyone. Regardless of your income, gender, sexual orientation, race, and other characteristics, we all must stay informed about current laws. Abortion is essential.");
```
Comment (catlynbowles) [inline cypress/e2e/aboutview.cy.js:34]: I love this language you used in the inclusivity piece. Great work, Irmak 🤌🏼

File: src/Views/AboutView/AboutView.js [hunk 3]
```diff
@@ -59,6 +59,10 @@ const AboutView = () => {
...
+        <p>The data on this website was last updated on September 6, 2022.</p>
+        <p>For most up to date information, please go <a href="https://www.abortionfinder.org">here</a>.</p>
+      </div>
```
Comment (catlynbowles) [inline src/Views/AboutView/AboutView.js:65]: Agreed, this was a great idea.

**Review by catlynbowles — APPROVED** [review]

**Pull request merged**